src/components/CreateProject/index.tsx

```diff
@@ -0,0 +1,190 @@
+import React, { useEffect, useState } from 'react';
+import Breadcrumb from 'components/Breadcrumb';
+import { Project, ProjectStatus, Role, User } from 'utils/types';
+import { USERS } from 'fakeData/users';
+import './index.scss';
+import { useCreateProjectForm } from './useCreateProjectForm';
+import { useHistory } from 'react-router-dom';
+import AddMember from './AddMember';
+
+function CreateProject() {
+   const history = useHistory();
+   const [listUsers, setListUsers] = useState<User[]>([...USERS]);
+   const [listUsersInProject, setListUsersInProject] = useState<User[]>([]);
+
+   const [formData, setFormData] = useState<Project>({
+      id: 1,
+      name: '',
+      description: '',
+      client: '',
+      status: ProjectStatus.Pending,
+   });
```

Comment:
Tạo type riêng để create Project nhé

```suggestion
   const [formData, setFormData] = useState<ProjectFormData>({
      id: 1,
      name: '',
      description: '',
      client: '',
      status: ProjectStatus.Pending,
      members: [],
      pm: null
   });
```